applications/StructuralMechanicsApplication/tests/explicit_tests/explicit_solid_beam_test_parameters.json

```diff
@@ -69,7 +69,7 @@
                     "model_part_name" : "Check_ModelPart",
                     "variable_name"   : "DISPLACEMENT_Y",
                     "interval"        : [0.0, "End"],
-                    "value"           : "-3.28e-5*(1.0 - cos((3.14159/0.0049)*t))",
+                    "value"           : "-3.28e-5*(1.0 - cos((3.14159/0.0049)*(t - 1.0e-6)))",
```

Comment:
didn't you recently fix this such that specifying it (t-dt) is no longer necessary?

Reply:
Yes, that why this delta is missing